Makefile

```diff
@@ -34,7 +40,7 @@ go/deps:
 .PHONY: go/bin
 go/bin: go/deps
 	mkdir -p ./bin
-	go build -o bin/ ./cmd/parca
+	go build $(SANITIZERS) -tags purego -o bin/ ./cmd/parca
```

Comment:
Could we please document which package(s) use `purego` tag? AFAIK it's not a standard build tag.

Reply:
My bad, this was not supposed to go here, was using it locally to workaround avx2 compatibility in parquet-go